cuckoo/cuckoo_result.py

```diff
@@ -20,8 +20,8 @@
 
 al_log.init_logging('service.cuckoo.cuckoo_result')
 log = getLogger('assemblyline.service.cuckoo.cuckoo_result')
-# Remove the part of the regex that looks to match the entire line
-URL_REGEX = re.compile(FULL_URI.lstrip("^").rstrip("$"))
+# Remove the part of the regex that looks to match the entire line. Create non-capturing group.
+URL_REGEX = re.compile(FULL_URI.lstrip("^").rstrip("$").replace("(", "(?:", 1))
```

Comment:
Having a cuckoo specific regex seems like it would be less fragile than accumulating differences from the odm validator